prow/spyglass/README.md

```diff
@@ -25,6 +25,15 @@ A general Spyglass query will proceed as follows:
   a blob of HTML.
 - Views (HTML) are inserted asynchronously as viewers return.
 
+# Available views
+
+Spyglass currently exposes the following views:
```

Comment:
job-history and pr-history aren't really part of spyglass.

Reply:
I suppose that makes sense? Because otherwise they're not linked anywhere?

Flag seems kinda overloaded to me but it is what it is.